doc/sphinx/source/conf.py

```diff
@@ -0,0 +1,38 @@
+# Configuration file for the Sphinx documentation builder.
+#
+# For the full list of built-in configuration values, see the documentation:
+# https://www.sphinx-doc.org/en/master/usage/configuration.html
+import os
+import sys
+
+# -- Project information -----------------------------------------------------
+# https://www.sphinx-doc.org/en/master/usage/configuration.html#project-information
+
+project = "Neo4j Graph Data Science Python Client"
+copyright = "2023, Neo4j"
+author = "Neo4j"
+release = "2023"
+
+# -- General configuration ---------------------------------------------------
+# https://www.sphinx-doc.org/en/master/usage/configuration.html#general-configuration
+
+# Add the root of the project to the path so that Sphinx can find the Python sources
+sys.path.append(os.path.join(os.path.dirname(__file__), "..", "..", ".."))
+
+extensions = [
+    "sphinx.ext.autodoc",
+    "sphinx.ext.napoleon",
+    # 'sphinx.ext.autosummary',
+]
+
+# napoleon_use_rtype = False
```

Comment:
looking at the return types did you try to not just return `DataFrame` but include the package name `pandas.DataFrame` or something?

Reply:
No, we could try it. For the py::function in .rst that can just be hardcoded, for the autofunction docstrings it will need a bit more effort.
But I think `DataFrame` should be clear enough?

Reply:
for this case you are probably right. Thought it could be nice to clarify, but I also did not find another reference doc